static/planttracer.js

```diff
@@ -319,7 +326,7 @@ function set_property(user_id, movie_id, property, value)
             if (data.error!=false){
                 $('#message').html('error: '+data.message);
             } else {
-                list_ready_function();
+                list_movies();
```

Comment:
Yeah, that's odd. I very recently changed this to list_ready_function() so again we have a merge-back fail.